Common/include/option_structure.hpp

```diff
@@ -958,13 +958,55 @@ inline TURB_FAMILY TurbModelFamily(TURB_MODEL model) {
   return TURB_FAMILY::NONE;
 }
 
+/*!
+ * \brief SST Options
+ */
+enum class SST_OPTIONS {
+  NONE,        /*!< \brief No SST Turb model. */
+  V1994,       /*!< \brief 1994 Menter k-w SST model. */
+  V2003,       /*!< \brief 2003 Menter k-w SST model. */
+  MODIFIED,    /*!< \brief Modified k-w SST model. */
+  SUST,        /*!< \brief Menter k-w SST model with sustaining terms. */
+  VORTICITY,   /*!< \brief Menter k-w SST model with vorticity production terms. */
+  KL,          /*!< \brief Menter k-w SST model with Kato-Launder production terms. */
+  UNCERTAINTY, /*!< \brief Menter k-w SST model with uncertainty quantification modifications. */
+  RC,          /*!< \brief Menter k-w SST model with rotation/curvature corrections. */
+};
+static const MapType<std::string, SST_OPTIONS> SST_Options_Map = {
+  MakePair("NONE", SST_OPTIONS::NONE)
+  MakePair("V1994m", SST_OPTIONS::V1994)
+  MakePair("V2003m", SST_OPTIONS::V2003)
```

Comment:
If you add "m" also to the enum you can then convert internally to 1994/2003 + the "m" boolean.
```suggestion
  MakePair("V1994m", SST_OPTIONS::V1994m)
  MakePair("V2003m", SST_OPTIONS::V2003m)
```